pkg/api/api.go

```diff
@@ -160,3 +162,13 @@ func (a *API) isAdmin(c *gin.Context) bool {
 
 	return session.IsAdmin
 }
+
+func (a *API) fetchAdminGroupID(){
+	id, err:= a.azureClient.GetGroupID(a.adminGroupMail)
+	if err!= nil{
+		a.log.WithError(err).Error("retrieve admin group id")
+		a.adminGroupID = "INVALID_ADMIN_GROUP_ID_CHECK_AAD_STATUS"
+		return		
```

Comment:
Ønsker vi at appen skal kjøres opp uten at admin-oppsettet er riktig/fungerende?

Reply:
I dont know, it is my assumption

Reply:
I think I'm leaning towards considering this a fatal error. If there's a problem interacting with AAD we will probably not be able to login users anyway.

Reply:
agree